script/Ciel/Day 9/09_01_CIEL09_1.txt

```diff
@@ -11,37 +11,38 @@
 [sha:8b06cb61ac00284736810d72c773b9f9026ba694]{
 -- Page 1, Offset 22140.
 -- バネじかけのカラクリにでもなったように、上半身がベッドから跳ね起きた。
-My upper body springs up from the bed like a coiled spring.
+I suddenly bolt upright in bed as through my torso were spring-loaded.
 }
 [sha:c58ae5db6286fdc4bede7c8348a5e556580c3df5]{
 -- Page 2, Offset 22141.
 -- 顔を上に向け、のど元までせり上がった嘔吐感を飲み下す。
-I raise my head, and the nausea that had risen up to my throat subsides.
+I tilt my head back, fighting to suppress the nausea that's risen in my throat.
 }
 [sha:0918fe1ec0aec082b1bc12a2aecd87101d7f30e8]{
 -- Page 2, Offset 22142.
 -- 全身にはぬるりとした赤い感触。
-My whole body feels like it's awash in sticky red.
+// Ah yes, everyone's favourite tactile sensation, 'red'
+A slimy red sensation clings to every surface of my body.
 }
 [sha:cf286986702d281fe4ce0dbd48d811bc146b5b6a]{
 -- Page 2, Offset 22143.
 -- よほど<魘|うな>されたのか大量の汗をかいている。
-My whole body is drenched in sweat, perhaps because I had a horrible nightmare.
+I must have been having a terrible nightmare. I'm absolutely drenched in sweat.
```

Comment:
It's probably better to render <魘|うな>された as something more like "I must've tossed a lot in my sleep" or something like that, since the next lines continues as ―――何か、良くない夢を、見たようだ。